evm/src/cpu/kernel/asm/hash/blake2b/util.asm

```diff
@@ -0,0 +1,47 @@
+// Load a 64-bit word from kernel general memory.
```

Comment:
I'd put this in `core.asm` next to `mload_kernel_u32` and give it a neutral name